org.dharkael.Flameshot.yaml

```diff
@@ -0,0 +1,36 @@
+app-id: org.dharkael.Flameshot
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: flameshot
+rename-icon: flameshot
+rename-appdata-file: flameshot.appdata.xml
+rename-desktop-file: flameshot.desktop
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host # write output files
+  - --filesystem=xdg-config/kdeglobals:ro # gives application access to kdeglobals
+  - --talk-name=com.canonical.AppMenu.Registrar # gives application access to dbus menu
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
+  - --share=network
+  - --env=PATH=/usr/bin:/app/bin:/app/bin/x86_64-linux
+  - --env=LC_ALL=C 
+build-options:
+  cflags: -O2
+  cxxflags: -O2
+  env:
+    LD_LIBRARY_PATH: /app/lib:/app/lib64
```

Comment:
Build system should be fixed to install files in `/app/lib`.

Reply:
removed